release/README.md

```diff
@@ -0,0 +1,32 @@
+Use Cases:
+1. Fork a current deployment on any network, given a config file
+    * Use - simulating upgrades/changes to a current deployment
+2. Deploy entire system from the current branch to any network
+    * Use - integration tests (mainly want to deploy locally)
+3. Easily deploy/upgrade _specific_ contracts on any ENV
+    * Use - writing deploy/upgrade scripts
+    * Note: this should also update that env's config with new addresses
+
+## Mock Command - Release/Upgrade Scripting
+
+```
+make release preprod
+```
+
+* Generates a script file (`preprod-<GIT_COMMIT_HEAD>.s.sol`) that automatically loads the config for the `preprod` environment
+* `make release preprod --test preprod-<GIT_COMMIT_HEAD>.s.sol`: 
```

Comment:
I like to use the specific release tag vs commit hash for more clarity but like the idea nonetheless

Reply:
i like that but i'm kinda forseeing a scenario where we don't have a release tagged yet